libraries/chain/asset_evaluator.cpp

```diff
@@ -375,12 +370,6 @@ void_result asset_update_issuer_evaluator::do_evaluate(const asset_update_issuer
               "Incorrect issuer for asset! (${o.issuer} != ${a.issuer})",
               ("o.issuer", o.issuer)("a.issuer", a.issuer) );
 
-   if( d.head_block_time() < HARDFORK_CORE_199_TIME )
-   {
-      // TODO: remove after HARDFORK_CORE_199_TIME has passed
-      FC_ASSERT(false, "Not allowed until hardfork 199");
-   }
-
    return void_result();
 } FC_CAPTURE_AND_RETHROW((o)) }
 
```

Comment:
Please replace the TODO comment on lines 479/609 with an explanation why it can't be removed.